tests/test_integration.py

```diff
@@ -77,6 +77,7 @@ def setUpClass(cls):
 
     def test_update_schemas(self):
         for model in db.Model._decl_class_registry.values():
+            print(model)
```

Comment:
Happy to take it out if it is too much of a style violation. 

We might want to keep it, because then the test tells you what model it fails at so you know what you need to add data to. I have added that print statement in at least 4 times now and it only prints if the test fails.